pages/manager/EmailHandler.inc.php

```diff
@@ -210,7 +210,188 @@ function enableEmail($args) {
 
 		Request::redirect(null, null, 'emails');
 	}
-
+	
+	/**
+	 * Export the selected email templates as XML
+	 * @param $args array
+	 * @@param $request PKPRequest
+	 */
+	function exportEmails($args, $request) {
```

Comment:
Be careful not to omit the calls to $this->validate() -- that's what checks that the user is a Journal Manager. Without it, anyone can access the new functions.